cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -37,6 +37,8 @@ spec:
     metadata:
       labels:
         app: metadata-agent
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
```

Comment:
Also add it to the `metadata-agent-cluster-level`?

Reply:
Thanks for the catch! Added now.